flow_screen_components/datatableFSC/force-app/main/default/aura/datatableFSC/datatableFSC.cmp

```diff
@@ -15,15 +18,17 @@
         Parameters for Column Width and Alignment
         Optional Checkboxes or Radio Buttons
         Pre-selected Rows
+        Editable Columns
 
     To change the predefined Objects, edit the Object names in datatableFSC.cmp and in datatableFSC.design
 
 -->
 
 <aura:component implements="lightning:availableForFlowScreens">
 
-	<aura:attribute name="obj" type="String" />
-	<aura:attribute name="mydata" type="SObject[]"/>
+    <aura:attribute name="obj" type="String" />
+    <aura:attribute name="mydata" type="SObject[]"/>
+    <aura:attribute name="budata" type="SObject[]"/>
```

Comment:
perhaps a more evocative attribute name? This is a draft of the data that includes inline changes?  let's add a comment too